releases/links_v2.0.0/src/InputParser.hpp

```diff
@@ -0,0 +1,248 @@
+#ifndef LINKS_INPUTPARSER_HPP
+#define LINKS_INPUTPARSER_HPP
+
+#include <vector>
+#include <queue>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <getopt.h>
+#include <sstream>
+
+//Globals
+#define BASE_TEN 10
+std::string progname = "LINKS";
+std::string version = "2.0.0";
+
+class InputParser {
+    private:
+    std::vector <std::string> tokens;
+    int argc;
+    int c;
+    int optindex = 0;
+	int help = 0;
```

Comment:
I would clang-format this code, as the indentation is wrong. It's not a big deal for this release, though.